packages/e2e-tests/src/puppeteerEnvironment.js

```diff
@@ -41,6 +41,11 @@ class PuppeteerEnvironment extends OriginalEnvironment {
   async handleTestEvent(event, state) {
     if (event.name === 'test_fn_failure') {
       const testName = `${state.currentlyRunningTest.parent.name}  ${state.currentlyRunningTest.name}`;
+      
+      const data = await this.global.page.screenshot({ type: 'jpeg', encoding: 'base64', quality: 1 	});
+      // eslint-disable-next-line no-console
+      console.log(`data:image/jpeg;base64,${data}`);
```

Comment:
Found this little trick here  - to output a base64 "image" to the console (in some cases we're not seeing screens captured in the artifacts package )
https://www.mixmax.com/engineering/maintaining-an-end-to-end-integration-test-suite

Testing this --- the timeouts look like we're getting stuck here

![download](https://user-images.githubusercontent.com/62242/178078812-4744e2b4-3fec-424d-a74e-276b91b71fe3.jpg)